pom.xml

```diff
@@ -349,16 +349,9 @@
 
          <!-- Required for: JMS Injection -->
          <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
-            <version>1.2</version>
-            <!-- License: Apache 2.0 -->
-         </dependency>
-         <dependency>
-            <groupId>javax.inject</groupId>
-            <artifactId>javax.inject</artifactId>
-            <version>1</version>
-            <!-- License: Apache 2.0 -->
+            <groupId>org.apache.geronimo.specs</groupId>
+            <artifactId>geronimo-atinject_1.0_spec</artifactId>
+            <version>1.0</version>
```

Comment:
JMS 2 defines injection points.  The way it ended up being phrased, its the app server's realm of responsibility to provide those injection points.  OpenMQ for example provides a CDI module.  HornetQ/Artemis do not as of now.  Wildfly's JMS integration module takes care of this, similarly TomEE does the same.

It turns out this is needed for the link you emailed earlier - the integration test for Karaf/OSGi.